lib/llm/src/migration.rs

```diff
@@ -133,6 +133,10 @@ impl RetryManager {
             self.retries_left -= 1;
             let request = Context::with_id(self.request.clone(), self.context.id().to_string());
             self.context.link_child(request.context());
+            if self.context.is_stopped() || self.context.is_killed() {
+                tracing::debug!("Abort creating new stream after context is stopped or killed");
+                return Err(Error::msg("Context is stopped or killed"));
```

Comment:
Can we add any kind of context or request ID in the error message here to make it easier to associate with other information in the logs?

Reply:
Yes, added context id to the error message.